src/Command/PluginLoadCommand.php

```diff
@@ -69,6 +71,15 @@ public function execute(Arguments $args, ConsoleIo $io): ?int
             return static::CODE_ERROR;
         }
 
+        try {
+            Plugin::getCollection()->findPath($plugin);
+        } catch (MissingPluginException $e) {
+            $this->io->err('Could not find plugin <info>NopeNotThere</info>');
```

Comment:
Need to use `$plugin`  instead of `NopeNotThere` in error message :slightly_smiling_face:.

Reply:
```suggestion
            $this->io->err($e->getMessage());
```

Reply:
I was pretty tired when I tried to do this and it shows. Sorry about that.